src/main.zig

```diff
@@ -46,6 +47,11 @@ pub fn main() !void {
     const query = try std.ascii.allocLowerString(allocator, args.query orelse "");
     defer allocator.free(query);
 
+    const regexQuery = mvzr.compile(query);
+    if (regexQuery == null) {
+        return std.io.getStdErr().writer().print("Invalid regular expression: parsing \"{s}\".", .{query});
+    }
```

Comment:
```suggestion
    const regexQuery = mvzr.compile(query) orelse
        return std.io.getStdErr().writer().print("Invalid regular expression: parsing \"{s}\".", .{query});
```

Removes the need for `.?` lower :)

Reply:
Thank you for helping me learn Zig concepts! ❤️